Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocListImplementation/Rendering/StubColumnRenderer.cs

```diff
@@ -31,24 +31,19 @@ public StubColumnRenderer (IResourceUrlFactory resourceUrlFactory)
     {
     }
 
-    protected override void RenderTitleCell (BocColumnRenderingContext<StubColumnDefinition> renderingContext, in BocTitleCellRenderArguments arguments)
```

Comment:
The StubColumnRenderer can no longer derive from ColumnRendererbase. Instead, we need to implement completely. Otherwise, we no render more stuff than before. 

Please check for red test, if no tests are red, a very compact implementation of ColumnDeclaration will suffice (i.e. just `<col />`)